packages/toolkit/types/index.ts

```diff
@@ -111,11 +111,26 @@ export type CreativeWorkAssertion = Assertion<
     url?: string;
   }
 >;
+interface Reference {
+  uri: string;
+}
+
+interface Resource {
+  reference: Reference;
+}
+
+export declare type C2paAssetRef = Assertion<
```

Comment:
```suggestion
export type C2paAssetRef = Assertion<
```
I don't think we need `declare` here.